style.css

```diff
@@ -1,3 +1,172 @@
 body {
+  margin: 0 auto;
+  font-family: arial, 'Share Tech Mono', monospace;
+  font-size: 10pt;
+  color: #fff;
+  background: #2b2b2b;
+  background: url("http://i.imgur.com/a6yVVuG.jpg");
+  background-position: top center;
+  background-size: 100vw;
+  background-repeat: no-repeat;
+}
+
+
+main {
+margin: 2rem auto;
+width: 100%;
+overflow: auto;
+display: inline-block;
+}
+
+.main--cell {
+margin: 0 auto;
+overflow: auto;
+width: 90%;
+}
+
+.character--cell {
+  /*background: blue;*/
+  width: calc(100% / 3 - 1rem - 1rem);
+  margin: 1rem 1rem;
+  position: fixed;
+  bottom: 4rem;
+  left: 1rem;
+  padding-left: 2rem;
+  overflow: auto;
+  display: block;
+  transform: rotate(-5deg);
+}
+
+.character__content {
+width: auto;
+padding-bottom: 2.5rem;
+overflow: auto;
```

Comment:
Here `overflow: auto` is making a scrollbar when mouses are plugged in. It's hard to see and debug sometimes. If you have a set width or height and don't want a scrollbar to possibly show up, use `overflow: hidden`